hoomd/md/ForceComposite.cc

```diff
@@ -1048,6 +1054,47 @@ void ForceComposite::updateCompositeParticles(uint64_t timestep)
         }
     }
 
+void ForceComposite::findRigidCenters()
+    {
+    ArrayHandle<unsigned int> h_tag(m_pdata->getTags(), access_location::host, access_mode::read);
+    ArrayHandle<unsigned int> h_rtag(m_pdata->getRTags(), access_location::host, access_mode::read);
+    ArrayHandle<unsigned int> h_body(m_pdata->getBodies(),
+                                     access_location::host,
+                                     access_mode::read);
+
+    m_rigid_center.resize(m_pdata->getN() + m_pdata->getNGhosts());
+    m_rigid_center.zeroFill();
+    m_lookup_center.resize(m_pdata->getN() + m_pdata->getNGhosts());
+    m_lookup_center.zeroFill();
```

Comment:
I don't think you can fill here and then overwrite below. However, since you will loop over everything anyway, let's restructure the code so this fill is not needed.
```suggestion
    m_lookup_center.resize(m_pdata->getN() + m_pdata->getNGhosts());
```